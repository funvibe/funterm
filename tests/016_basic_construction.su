lua.empty = <<>>
lua.single_byte = <<42>>
lua.three_bytes = <<1, 2, 3>>
lua.from_string = <<"hello">>

lua.print("Empty length:", lua.len(lua.empty))
lua.print("Single byte:", lua.single_byte[0])
lua.print("Three bytes:", lua.three_bytes[0], lua.three_bytes[1], lua.three_bytes[2])
lua.print("String as bytes:", lua.from_string[0], lua.from_string[1])