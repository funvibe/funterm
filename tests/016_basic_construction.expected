Empty length: 0
Single byte: *
Three bytes: 1 2 3
String as bytes: h e